.github/workflows/core.yml

```diff
@@ -346,12 +341,7 @@ jobs:
       fail-fast: false
       matrix:
         python: [ 3.9 ]
-        java: [ 8, 11 ]
-        include:
-          - python: 3.7
-            java: 8
-          - python: 3.8
-            java: 8
```

Comment:
this silently drops test for Python 3.7 and 3.8, if this is intended, please at least mention that in PR title/desc

Reply:
The description has been adjusted.

Reply:
> ... also drops python 3.8 & 3.9

typo? should be 3.7 & 3.8

Reply:
Definitely, thank you for your close look.
Adjusted.